src/main/java/net/mcreator/ui/modgui/DimensionGUI.java

```diff
@@ -350,10 +366,15 @@ private void updatePortalElements() {
 	}
 
 	@Override protected AggregatedValidationResult validatePage(int page) {
-		if (page == 0)
+		if (page == 0) {
+			if ((int) minHeight.getValue() % 16 != 0 || (int) height.getValue() % 16 != 0)
+				return new AggregatedValidationResult.FAIL(L10N.t("elementgui.dimension.error_invalid_height_values"));
+			else if ((int) logicalHeight.getValue() > (int) height.getValue())
+				return new AggregatedValidationResult.FAIL(L10N.t("elementgui.dimension.error_logical_bigger_height"));
 			return new AggregatedValidationResult(page2group);

```

Comment:
The way you implemented this, the user will need to fix min & total height values to then fix the logical height value to even see the other errors on this page. Just speaking aloud, don't really know what to do with this, maybe Klemen would help you.

Reply:
I'm aware of this problem. However, the PageGroup class doesn't support conditions... I would need to create a new JSPinner class to support this.